README.md

```diff
@@ -1,2 +1,28 @@
-# hw01
-Repository for first CFSS homework assignment
+---
+title: "Fran's first markdown" 
+author: "Frances Mendes Levitin"
+---
+
+
+# hw01: Repository for first CFSS homework assignment
+
+## Who
+My name is Frances Mendes Levitin.  I am a first year student in the *Department of Visual Arts* (DoVA) at University of Chicago. To see more art go to by website by clicking this [link](http://45.55.61.6:3000/).  
+
+### Art Practice Activities
+* Scavenging   
+* Programing  
+* Hacking  
+* Live Audio/Visual performance
+* Painting
+
+
+**ANALOG VIDEO MIXER I hope to build myself**
+![](visualCortex.png)
```

Comment:
Will be a good idea to create a sub folder for images in the future for repository organization, when we submit include more images in our submission